src/content/getting_started/architecture/gateway-engine/_index.en.md

```diff
@@ -63,3 +63,12 @@ Gateway node in the cluster.
 
 The impact on datapath for various scenarios in a kind setup are captured in the
 following [spreadsheet](https://docs.google.com/spreadsheets/d/1JsXsyRDDXkp6t55Gm-NP5EggWTyYi2yo27pyuDYwlpc/edit#gid=0).
+
+#### Gateway Health Check
+
+The Gateway Engine continuously monitors the health of connected clusters.
+It periodically pings each cluster and collects statistics including basic connectivity,
+round trip time (RTT) and average latency. This information is updated in the 'Gateway'
+resource. Whenever the Gateway Engine detects an unhealthy cluster, its connection status
```

Comment:
Can we define what "unhealthy cluster" means? We also wanted to expose the timer values so that this is configurable, and have the option to turn the feature off?

Reply:
Done.